web-console/src/routes/(authenticated)/(pipelines)/create/+page.ts

```diff
@@ -1,31 +1,33 @@
 import { goto } from '$app/navigation'
 import { base } from '$app/paths'
 import { useTryPipeline } from '$lib/compositions/pipelines/useTryPipeline'
-import type { PipelineDescr } from '$lib/services/manager'
+import type { Demo } from '$lib/services/manager'
 import { getDemos } from '$lib/services/pipelineManager'
 
 export async function load({ url, parent }) {
   await parent()
-  const pipeline: PipelineDescr | undefined = await (() => {
+  const pipeline: Omit<Demo, 'title'> | undefined = await (() => {
```

Comment:
This extra service should be separate from the demos (which are a special construct for that endpoint), why is that data structure used here?

Reply:
Semantically it makes sense - you essentially create a demo - either looked up by name from a list of known demos, or constructed from URL argunents